sdks/java/core/src/main/java/org/apache/beam/sdk/util/IdentityWindowFn.java

```diff
@@ -55,25 +56,21 @@
    * these windows.
    */
   private final Coder<BoundedWindow> coder;
-  private final boolean assignsToSingleWindow;
 
-  public IdentityWindowFn(Coder<? extends BoundedWindow> coder, boolean assignsToSingleWindow) {
+  public IdentityWindowFn(Coder<? extends BoundedWindow> coder) {
     // Safe because it is only used privately here.
     // At every point where a window is returned or accepted, it has been provided
     // by priorWindowFn, so it is of the expected type.
```

Comment:
There's two references to `priorWindowFn`. As it is it looks like some field but it should probably be "prior WindowFn"?